setup.py

```diff
@@ -102,6 +102,6 @@
                         'scikit-bio >= 0.2.3, < 0.3.0', 'wtforms == 2.0.1',
                         'qiime >= 1.9.0, < 1.10.0', 'moi >= 0.2.0, < 0.3.0',
                         'sphinx-bootstrap-theme', 'Sphinx >= 1.2.2',
-                        'gitpython'],
+                        'gitpython', 'seaborn'],
```

Comment:
In general I would agree with you but seaborn is something that all the plugins showing plots are going to use. Thus, IMOO installing it in the main install is fine.